tests/test_dataset_report.py

```diff
@@ -4,23 +4,11 @@
 import numpy
 import pandas
 import pytest
+from pandas import testing
 
 from analysis import dataset_report
 
 
-@pytest.fixture
-def dataframe():
-    return pandas.DataFrame(
-        {
-            "patient_id": [1],
-            "is_registered": [True],
-            "is_dead": [False],
-            "stp_code": ["STP0"],
-            "has_sbp_event": [True],
-        }
-    )
-
-
```

Comment:
I'm being a bit cheeky here and not putting this in a separate commit. Hopefully that's okay 🙂 We stopped using the `dataframe` fixture several commits ago.